docs/source/index.rst

```diff
@@ -11,14 +11,26 @@ with Numba, we suggest you start with the :doc:`User manual <user/index>`.
 
 
 .. toctree::
-   :numbered:
```

Comment:
I don't think this will be missed, having built the docs locally, I cannot think of a time when I've needed a section number apart from to convey information to another person about something that's broken when debugging doc builds. Also having enumerations like `2.1.3.4` doesn't seem particularly useful.